File: 3226-minimum-number-game/minimum-number-game.java
class Solution {
    public int[] numberGame(int[] nums) {
        int res[] = new int[nums.length] ;
        Arrays.sort(nums);
        int k=0;
        for(int i=0;i<nums.length-1;i+=2)
        {
            res[k++]=nums[i+1];
            res[k++]=nums[i];
        }
        return res;
    }
}

File: 3226-minimum-number-game/minimum-number-game.cpp
class Solution {
public:
    vector<int> numberGame(vector<int>& nums) {
        vector<int>res;
        sort(nums.begin(),nums.end());
        for(int i=0;i<nums.size()-1;i+=2)
        {
            res.push_back(nums[i+1]);
            res.push_back(nums[i]);
        }
        return res;
    }
};